src/opcode.rs

```diff
@@ -227,23 +227,36 @@ opcode! {
 }
 
 opcode! {
-    /// Read from pre-mapped buffers that have been previously registered with
+    /// Read from a file into a fixed buffer that has been previously registered with
     /// [`Submitter::register_buffers`](crate::Submitter::register_buffers).
     ///
+    /// The operation designated by this `opcode` will read data from a file at a specific offset
```

Comment:
In my opinion there is no need to add too much description, something as simple as the manpage is fine.

Reply:
The only reason I added so much documentation is because when I first tried to use this crate I had zero clue how to interpret this and I spent a long time trying to figure out how these opcodes were related to the io_uring C counterparts. However, if there is a better man page, that would certainly be better than this.